01-filedir.md

```diff
@@ -336,7 +336,10 @@ As you can see,
 it also displays another special directory that's just called `.`,
 which means "the current working directory".
 It may seem redundant to have a name for it,
-but we'll see some uses for it soon.
+but we'll see some uses for it soon.  
+Finally, we also see a file called `.bash_profile`. This file usually contains settings to customize the shell (terminal). There may also be similar files called `.bashrc` or `.bash_login`. For this lesson material it does not contain any settings.
```

Comment:
Looks good. I suggest adding to: For this lesson material the dotfile, .bash_profile, does not contain any workshop settings.

I am afraid 'workshop settings' may be confusing...

The dotfile, .bash_profile would be hidden when executing an ls command without the -a "show all" option.

The whole point of adding is to show that adding '-a' to 'ls' shows this file. I feel adding this sentence is redundant and unnecessary.